app/Http/Controllers/UserController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\UserUpdateRequest;
+use App\Models\User;
+use App\Services\UserService;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class UserController extends Controller
+{
+    public function __construct(UserService $userService)
+    {
+        $this->userService = $userService;
+    }
+
+    public function getUsers(): JsonResponse
+    {
+        $users = User::all();
+
+        return response()->json([
+            'users' => $users,
+            'message' => 'Users found',
+            'status' => 1
+        ]);
+    }
+
+
+    public function update(UserUpdateRequest $request)
+    {
+        $user = auth()->user();
```

Comment:
informative comment: We also can get user object from the request
`$user = $request->user()`

btw we should use another variable for the authorized user, for ex.:
$authUser